.gitignore

```
@@ -10,4 +10,7 @@ build/
 dist/
 *.egg-info
 
+# Secret
+/pybf/config.json
```

```suggestion
config.json
```
必ずしも`/pybf`内に作成するとは限らないので、ディレクトリに関係なく追跡対象から除外すべきかな

---

a07761e で修正しましたー！